app/views/LocationShare/LocationPreviewModal.tsx

```
@@ -0,0 +1,172 @@
+import React, { useState, useMemo, useEffect, useRef } from 'react';
+import { View, Text, TouchableOpacity, StyleSheet, Alert, Linking } from 'react-native';
+import { Image as ExpoImage } from 'expo-image';
+
+import Navigation from '../../lib/navigation/appNavigation';
+import { staticMapUrl, MapProviderName, providerLabel, mapsDeepLink } from './services/mapProviders';
+import { sendMessage } from '../../lib/methods/sendMessage';
+import { useAppSelector } from '../../lib/hooks';
+import { getUserSelector } from '../../selectors/login';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)